bokeh/models/annotations.py

```diff
@@ -528,7 +528,8 @@ class Band(Annotation):
     """)
 
     dimension = Enum(Dimension, default='height', help="""
-    The direction of the band.
+    The direction of the band can be specified by setting this property 
+    to ``height`` (``y`` direction) or ``dimension`` (``x`` direction).
```

Comment:
typo, should be "or ``width``" (same comment for others as well)

Reply:
Will change